utils/maintenance.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
+
+# file: maintenence.sh
+# description: performs all cyclus project maintenence tasks for a release 
+
+set -e
+
+usage() {
+
+cat <<EOF
+
+Script to perform maintenance tasks related to the release of Cyclus and Cycamore.
+
+usage: $0 -v|--version <version_string> [-a|--abi_chk] [-d|--nuc_data] [-p|--pyne] [-s|--stub]
+       $0 -v|--version <version_string> -r|--release
+
+    -v|--version  : provide a version string for these tasks in the form X.Y.Z (REQUIRED)
+    -a|--abi_chk  : perform test for ABI consistency
+    -d|--nuc_data : upload new nuc_data.h5
+    -p|--pyne     : update amalgamated PyNE
+    -r|--release  : perform ALL tasks for release (same as -a -d -p -s)
+    -s|--stub     : update cycstub
+
+The following environment variables must be set for this script to function:
+
+     CYCLUS_DIR : Environment variable CYCLUS_DIR must be set to the cyclus repository directory.
+    CYCSTUB_DIR : Environment variable CYCSTUB_DIR must be set to the cycstub repository directory.
+
+The -d|--nuc_data option requires the rs.cred files found in the Accounts Metadata file on Google Drive.
+
+EOF
+
+}
+
+die() {
+    echo -n >&2 "ERROR: "
+    echo >&2 "$@"
+    usage
+    exit 1
+}
+
+
+update_pyne() {
+
```

Comment:
done

On Fri, May 8, 2015 at 5:56 PM, Anthony Scopatz <notifications@github.com>
wrote:

> In utils/maintenance.sh
> <https://github.com/cyclus/ciclus/pull/63#discussion_r29983392>:
>
> > +The -d|--nuc_data option requires the rs.cred files found in the Accounts Metadata file on Google Drive.
> > +
> > +EOF
> > +
> > +}
> > +
> > +die() {
> > +    echo -n >&2 "ERROR: "
> > +    echo >&2 "$@"
> > +    usage
> > +    exit 1
> > +}
> > +
> > +
> > +update_pyne() {
> > +
>
> Inconsistent whitespace. die() does not have this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/ciclus/pull/63/files#r29983392>.
>



-- 
Matthew Gidden, Ph.D.
Postdoctoral Associate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192